scopes/component/ui/version-dropdown/version-info/version-info.module.scss

```diff
@@ -4,41 +4,57 @@
   align-items: center;
   height: 40px;
   padding: 0 16px;
+  line-height: 24px;
 }
+
 .versionTimestamp {
   min-width: fit-content;
   margin-right: 2px;
   text-align: right;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
 }
+
 .versionUserAvatar {
+  flex: none;
   padding: 0px 8px;
 }
+
 .laneIcon {
   padding: 0px 4px;
 }
+
 .version {
   max-width: 70%;
   display: flex;
   flex-direction: row;
-  align-content: space-between;
   align-items: center;
+  justify-content: flex-start;
+  overflow: hidden;
 }
+
 .versionName {
   padding: 0px 8px;
-  &.tag {
-    min-width: fit-content;
-  }
-  &.snap {
-    min-width: 20%;
-  }
+  flex-grow: 1;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  min-width: fit-content;
 }
 
-.commitMessage,
-.emptyMessage {
-  min-width: 50%;
+.commitMessage {
+  flex-grow: 1;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
Same here, consider using: https://bit.cloud/teambit/design/ui/styles/ellipsis
You can use only the class from this component if you prefer.